apps/web/components/ProductRecommendedProducts/ProductRecommendedProducts.vue

```diff
@@ -1,12 +1,14 @@
 <template>
-  <p
+  <div class="text-2xl" v-if="preTitle">{{ preTitle }}</div>
+  <h2
     v-if="headline"
```

Comment:
```suggestion
    v-if="title"
```
Our naming should be consistent throughout all our blocks, so in this case `pretitle`, `title`, `subtitle`.